frontend/src/components/datasets/ExpandableTableDatasetComments.tsx

```diff
@@ -9,7 +9,7 @@ import { Dataset } from "../../models/frontend_models";
 export default function ExpandableTableDatasetComments(): NonNullable<TableProps<any>['expandable']> {
   return (
     {
-      columnTitle: <div>Comments</div>,
+      columnTitle: <CommentOutlined />,
```

Comment:
I personally prefer to keep text in the headers ... those icons are really not that helpful when it comes to understanding their meaning ...

Reply:
i have no quarrels with either or, although i would be interested to know how the icon approach will be received by the users. i can set it back to the string 'comments' if you are ok with it. 

(nowadays, thanks to fact that almost everyone has a smart phone with them and the main means of communicating on the some of these modern apps is to display icons)